apps/web/pages/api/trpc/features/[trpc].ts

```diff
@@ -1,4 +1,4 @@
 import { featureFlagRouter } from "@calcom/features/flags/server/router";
 import { createNextApiHandler } from "@calcom/trpc/server/createNextApiHandler";
 
-export default createNextApiHandler(featureFlagRouter);
+export default createNextApiHandler(featureFlagRouter, true, "features");
```

Comment:
We can now optionally namespace api handlers to distinguish them @sean-brydon @hariombalhara